src/tests/ftest/deployment/target_failure.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+
+timeout: 10M
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.0"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["0000:00:00.2"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+
+pool_size_ratio:
+  size: 50%
+  control_method: dmg
+pool_size_value:
+  scm_size: 100G
+  nvme_size: 300G
+  control_method: dmg
+
+container_wo_rf:
+  type: POSIX
+  control_method: daos
+container_with_rf:
+  type: POSIX
+  control_method: daos
+  properties: rf:1
+
+ior:
+  client_processes:
+    np: 4
```

Comment:
I would recommend bumping the np/ppn higher if possible, just to create more opportunities for race conditions. Or, at the very least, `ppn: 4` so clients processes will scale when more client nodes are given.

Reply:
I updated to use `ppn`, but how is it used? Now the mpirun command that runs IOR uses `-np 1`. I don't see `ppn` in the IOR parameters either.

Reply:
Ah, this is because you're creating your own `IorCommand`. I think whether you use `np` or `ppn` the command will use `-np 1` by default unless you assign the processes. This is what `IorTestBase.run_ior` does:
https://github.com/daos-stack/daos/blob/4c0917c7880133d38989cc69c35a52919e8905a1/src/tests/ftest/util/ior_test_base.py#L272-L276

Reply:
Fixed. Thanks.